jcheck/src/main/java/org/openjdk/skara/jcheck/CopyrightFormatConfiguration.java

```diff
@@ -29,20 +29,54 @@
 import java.util.regex.Pattern;
 
 public class CopyrightFormatConfiguration {
+
+    public static class CopyrightConfiguration {
```

Comment:
Did you consider making this a record? Looks like a good fit.

Reply:
Yes, it makes sense. I have made it a record